setup.py

```diff
@@ -37,4 +37,5 @@
     description="CLP FFI Python Interface",
     ext_modules=[ir],
     packages=["clp_ffi_py"],
+    version="0.0.1"
```

Comment:
Probably worth adding a comment in pyproject.toml that the packages and version specification also need to be specified in `setup.py` (and why)